scripts/test.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
```

Comment:
I know you weren't the one who created the file, but do you think we should make this script executable as we do with scripts in other repos?

Reply:
Generally speaking I'd prefer to limit the changes here in favor of getting the code in. I can make this one change, but outside of this please limit the asks to the PR's functionality.